### ISSUE TITLE
spacy_sklearn backend requiring mitie

### ISSUE THREAD

**Issue by shivkanthb** [golastmile/rasa_nlu#272]
**rasa NLU version** (e.g. `0.7.3`):
0.7.3

**Used backend / pipeline** (`mitie`, `spacy_sklearn`, ...):
spacy_sklearn

**Operating system** (windows, osx, ...):
osx

**Issue**:
Even when I try to train using the spacy_sklearn backend, system still looks for mitie. 
I get the following error when I try to run it
```
ImportError: No module named mitie
```

**Content of configuration file** (if used & relevant):
```
{
  "pipeline": "spacy_sklearn",
  "path" : "./models",
  "data" : "./data/start-stop.json"
}
```

**Comment by tmbo** [comment]
@shivkanthb this is fixed on master. But if you don't mind sharing the backtrace of the exception I can fix it in the 0.7.x versions as well.

**Comment by shivkanthb** [comment]
Sure. Ill use the master one for now. Thank you :) 
```
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/Users/shiv/dev/start_stop_rasa/venv_spacy/lib/python2.7/site-packages/rasa_nlu/train.py", line 72, in <module>
    do_train(config)
  File "/Users/shiv/dev/start_stop_rasa/venv_spacy/lib/python2.7/site-packages/rasa_nlu/train.py", line 60, in do_train
    trainer = create_trainer(config)
  File "/Users/shiv/dev/start_stop_rasa/venv_spacy/lib/python2.7/site-packages/rasa_nlu/train.py", line 29, in create_trainer
    from trainers.mitie_trainer import MITIETrainer
  File "/Users/shiv/dev/start_stop_rasa/venv_spacy/lib/python2.7/site-packages/rasa_nlu/trainers/mitie_trainer.py", line 4, in <module>
    from mitie import *
ImportError: No module named mitie
```

**Comment by tmbo** [comment]
Won't be fixed in 0.7. Nevertheless, I'll close this on as it is fixed in the current master which will be released soon.
